samples/README.md

```diff
@@ -0,0 +1,17 @@
+# Samples
+
+_To help you, we always try to add samples in order to facilitate Datamaintain adoption on your project_
```

Comment:
In my opinion, this calls for new issues, suggesting workflows that we could think about. It could be interesting to suggest an issue template for this and add a link here to issue creation.

Reply:
I don't really know if we want to spend time about that for now. If we see it is a need you will do it.